packages/tools/examples/annotationSelectionAndLocking/index.ts

```diff
@@ -147,7 +147,7 @@ async function run() {
       },
     ],
   });
-  toolGroup.setToolPassive(BidirectionalTool.toolName);
+  toolGroup.setToolPassive(RectangleROITool.toolName);
```

Comment:
Curious of why you changed bidirectional to rectangleROI?

Reply:
I shouldnt commit that at all.. it was just because of testing purpose